### DISCUSSION TITLE
Option for MYSQL_OPT_TLS_SNI_SERVERNAME

### DISCUSSION THREAD

**Discussion by VladoPortos** [PyMySQL/mysqlclient#661] [Q&A]
Hello,
Any idea how to use MYSQL_OPT_TLS_SNI_SERVERNAME  for mysql_options() ? Its mentioned that it was finally implemented here: https://bugs.mysql.com/bug.php?id=84849 but I'm not sure how to use it. (Or if its even implemented in library, I can't find any client that would support it )

**Comment by methane** [comment]
> Any idea how to use MYSQL_OPT_TLS_SNI_SERVERNAME for mysql_options() ?

Patch the source code to use MYSQL_OPT_TLS_SNI_SERVERNAME.
